src/views/Add User/FormAddUser.js

```diff
@@ -0,0 +1,371 @@
+import React, { useState } from "react";
+import { Form, Button, Container } from "react-bootstrap";
+
+function AddUserForm() {
+    const [userData, setUserData] = useState({
+        nik: "",
+        fullname: "",
+        email: "",
+        password: "",
+        level: "",
+        directorate: "",
+        divisionName: "",
+        unitName: "",
+        location: "",
+        roleName: "",
+        positionName: ""
+    });
+
+    const directorateOptions = [
+        "Direktorat Perdagangan dan Pengaturan Anggota Bursa",
+        "Direktorat Pengawasan Transaksi dan Informasi",
+        "Direktorat Pengawasan Transaksi dan Kepatuhan",
+        "Direktorat Teknologi Informasi dan Manajemen Risiko",
+        "Direktorat Penilaian Perusahaan",
+        "Direktorat Keuangan dan Sumber Daya Manusia",
+        "Direktorat Pengembangan",
+        "Direktorat Utama",
+    ];
```

Comment:
:wrench: Create JSON and import these options for readability. In the future, we need to fetch this from our database